### PR TITLE
Railgun aimer

### PR DESCRIPTION
**Pull request by covertcorvid**
## About The Pull Request
Adds a click-and-drag aiming beam to the overmap view when the gunner is in railgun fire mode.

## Why It's Good For The Game
Hopefully helps make combat more fun and improve survival chances.

## Changelog
:cl:
add: Added aiming tool for railgun
/:cl:

### CODE REVIEW

File: nsv13/code/modules/overmap/aim_helper.dm [hunk 1]
```diff
@@ -0,0 +1,107 @@
+/mob/living
+	var/obj/structure/overmap/gunning_ship = null //NSV13 - reference to overmap for aim helper
```
Comment (Kmc2000) [inline nsv13/code/modules/overmap/aim_helper.dm]: Not hugely sure how I feel about having another overmap reference further to get_overmap()

File: nsv13/code/modules/overmap/aim_helper.dm [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	P.color = "#99ff99"
+	var/turf/curloc = get_turf(src)
+	var/turf/targloc = get_turf(gunner.client.mouseObject)
+	if(!istype(targloc))
+		if(!istype(curloc))
+			return
+		targloc = get_turf_in_angle(lastangle, curloc, 10)
+	P.preparePixelProjectile(targloc, src, gunner.client.mouseParams, 0)
+	P.layer = BULLET_HOLE_LAYER
+	P.fire(lastangle)
```
Comment (Kmc2000) [inline nsv13/code/modules/overmap/aim_helper.dm]: This is disgusting but I can't think of a better way to do this.

**Comment by Kmc2000** [comment]
Is there code for actually firing the railgun here? I can't see any references to fire_railgun()

File: nsv13/code/modules/overmap/aim_helper.dm (hunk 1 above)
Comment (covertcorvid) [reply nsv13/code/modules/overmap/aim_helper.dm]: I got rid of it

**Comment by covertcorvid** [comment]
> Is there code for actually firing the railgun here? I can't see any references to fire_railgun()

It now fires the railgun

**Pull request merged**